### PR TITLE
Docs navigation using tabs

### PR DESCRIPTION
**Pull request by tpdorsey**
Updates docs configuration and minor associated edits to the text.

## Change Summary

* Enables navigation tabs (topic tabs in the header) to open more space for new topics
* Renames page TOC to "Page contents"
* Renames home page to "Welcome to pydantic," and reorganizes home page to focus on use case and examples first.
* Removes sponsors from home page
* Minor copyedits for clarity and consistency

tabbednav

## Checklist

* [ ] Unit tests for the changes exist
* [ ] Tests pass on CI and coverage remains at 100%
* [x] Documentation reflects the changes where applicable
* [ ] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)
* [ ] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ conda install pydantic -c conda-forge
 
 ## Compiled with Cython
```
Comment (samuelcolvin) [inline docs/install.md:18]: this whole section is going since we no longer compile with cython.

File: docs/index.md [hunk 2]
```diff
@@ -7,82 +7,13 @@
...
-And many more who kindly sponsor Samuel Colvin on [GitHub Sponsors](https://github.com/sponsors/samuelcolvin#sponsors).
-
-<script>
-  // randomize the order of sponsors
-  const ul = document.querySelector('.sponsors')
-  for (let i = ul.children.length; i >= 0; i--) {
-    ul.appendChild(ul.children[Math.random() * i | 0])
-  }
-</script>
+*pydantic* is the most widely used data validation library for Python.
```
Comment (samuelcolvin) [inline docs/index.md:10]: I wonder about changing this slightly odd convention of referring to pydantic in lower case italics.

I think "Pydantic" probably makes more sense.

File: docs/index.md [hunk 3]
```diff
@@ -137,51 +68,51 @@ So *pydantic* uses some cool new language features, but why should I actually go
 Hundreds of organisations and packages are using *pydantic*, including:
 
 [FastAPI](https://fastapi.tiangolo.com/)
-: a high performance API framework, easy to learn,
+: A high performance API framework, easy to learn,
   fast to code and ready for production, based on *pydantic* and Starlette.
 
 [Project Jupyter](https://jupyter.org/)
-: developers of the Jupyter notebook are using *pydantic*
+: Developers of the Jupyter notebook are using *pydantic*
```
Comment (samuelcolvin) [inline docs/index.md:75]: Let's talk about which packages to mention here - this list is very old and definitely needs to be updated.

**Review by samuelcolvin — COMMENTED** [review]
overall, I think looking good.

The `install.md` bit will conflict heavily with the PR I'm about to submit as that whole section is removed.

File: docs/index.md (hunk 2 above)
Comment (tpdorsey) [reply docs/index.md:10]: I think both uses are fine, but on board with standardizing on Pydantic except in code examples, as relevant

File: docs/install.md [hunk 4]
```diff
@@ -32,16 +32,18 @@ print('compiled:', pydantic.compiled)
 
 ### Performance vs package size trade-off
```
[the remark this answers is not in the document]
Comment (tpdorsey) [reply docs/install.md:33]: @samuelcolvin related to this:

Pydantic V1/Pydantic V2 or Pydantic 1/Pydantic 2(/Pydantic 3... etc)

My preference is for the latter (no V). What do you think?

File: docs/index.md (hunk 3 above)
Comment (tpdorsey) [reply docs/index.md:75]: Making this a separate task to follow up on

File: docs/install.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (samuelcolvin) [reply docs/install.md:33]: Again I don't care as long as w'ere consistent. I've talked about "Pydantic V2" until now, but happy to change.

**Comment by samuelcolvin** [comment]
this is great, thank you.

**Pull request merged**